lib/dfe/reference_data/itt/cycles.rb

```diff
@@ -108,19 +108,19 @@ def self.make_local_time(*args)
             apply_1_deadline: nil, # This deadline is not applicable for this cycle
             apply_2_deadline: make_local_time(2025, 9, 16, 18), # 3rd Tuesday in September. This is the final and only deadline to send ITT applications in the cycle
             provider_decision_deadline: make_local_time(2025, 9, 24, 23, 59, 59), # 1 week and a day after Apply 2 deadline
-            find_closes: make_local_time(2025, 10, 6, 23, 59, 59), # The evening before find opens in the new cycle
+            find_closes: make_local_time(2025, 9, 30, 23, 59, 59), # The evening before find opens in the new cycle
             non_working_days: {
               christmas: Date.new(2024, 12, 16)..Date.new(2025, 1, 3),
               easter: Date.new(2025, 4, 7)..Date.new(2025, 4, 21)
             }
           },
           '2025-2026' => {
-            find_opens: make_local_time(2025, 10, 7, 9), # First Tuesday of October
-            apply_opens: make_local_time(2025, 10, 14, 9), # Second Tuesday of October
+            find_opens: make_local_time(2025, 10, 1, 9), # Wednesday instead of Tuesday so the cycle opens the first week in October
+            apply_opens: make_local_time(2025, 10, 8, 9), # Second Wednesday of October
             apply_1_deadline: nil, # This deadline is not applicable for this cycle
-            apply_2_deadline: make_local_time(2026, 9, 22, 18), # 3rd Tuesday in September. This is the final and only deadline to send ITT applications in the cycle
-            provider_decision_deadline: make_local_time(2026, 9, 30, 23, 59, 59), # 1 week and a day after Apply 2 deadline
-            find_closes: make_local_time(2026, 10, 5, 23, 59, 59), # The evening before find opens in the new cycle
+            apply_2_deadline: make_local_time(2026, 9, 16, 18), # 3rd Wednesday in September. This is the final and only deadline to send ITT applications in the cycle
```

Comment:
Now apply_1 and apply_2 don't exist. Is there a plan to change this to just apply_deadline? Appreciate that this is a separate point to this PR

Reply:
In our code base, we just have apply_deadline now. It is a breaking change though if people are reference `apply_1` and `apply_2` in your data.

Reply:
ah might be one for Data Insights team to pick up on improving on our reports